openex-front/src/admin/components/assets/endpoints/Endpoints.tsx

```diff
@@ -274,13 +299,19 @@ const Endpoints = () => {
                     className={classes.bodyItem}
                     style={inlineStyles.asset_tags}
                   >
-                    <ItemTags variant="list" tags={endpoint.asset_tags} />
+                    <ItemTags variant="list" tags={endpoint.asset_tags}/>
+                  </div>
+                  <div
+                    className={classes.bodyItem}
+                    style={inlineStyles.asset_status}
+                  >
+                    {differenceInHours(new Date().toISOString(), endpoint.asset_last_seen) > MAX_ALIVE_HOURS ? <AssetStatus variant="list" status='Inactive'/> : <AssetStatus variant="list" status='Active'/>}
```

Comment:
You need to check if last_seen exist -> if not don't check the difference  just put inactive

Reply:
Done